pkg/apis/v1/nodeclaim_validation.go

```diff
@@ -68,74 +67,75 @@ func (in *NodeClaim) SupportedVerbs() []admissionregistrationv1.OperationType {
 }
 
 // Validate the NodeClaim
-func (in *NodeClaim) Validate(_ context.Context) (errs *apis.FieldError) {
-	return errs.Also(
-		apis.ValidateObjectMetadata(in).ViaField("metadata"),
-		in.Spec.validate().ViaField("spec"),
-	)
+func (in *NodeClaim) Validate(_ context.Context) error {
```

Comment:
Do we still need this function? AFIK this was only needed to be a Knative interface